source/includes/aws/_instances.md

```diff
@@ -282,7 +332,29 @@ Create a new instance in a given [environment](#administration-environments).
 | `vpcId` <br/>*string* | The id of the VPC in which the instance will be deployed. This must be specified if there is no default VPC in the selected region or if you decide to provide a subnet. |
 |`subnetId` <br/>*string* | The id of the subnet in which the instance will be deployed. If this is not specified, the instance will be deployed in one of the default subnets in the default VPC. The subnet specified must belong to the VPC specified.
 
-
+Attributes | &nbsp;
+------- | -----------
+`id`<br/>*string* | The task's ID.
+`status`<br/>*string* | The task's status.
+`created`<br/>*Date* | The task's creation date.
+`result`<br/>*Object* | The task result, containing details of the created instance.
+`result.`<br/>*string* | The administrator username created on the instance.
```

Comment:
You forgot the field.

Reply:
Thanks, removed [here](https://github.com/cloudops/cloudmc-api-docs/pull/562/commits/8528868240ff2508537c1542783f1c609757ba4a)